guides/common/assembly_preparing-environment-for-capsule-installation.adoc

```diff
@@ -7,6 +7,9 @@ ifdef::context[:parent-context: {context}]
 
 Review the following prerequisites before you install {SmartProxyServer}.
 
+// Supported Operating Systems
```

Comment:
the comment is outdated due to renaming the module. IMO all our files have great names, so comments are mostly useless.

```suggestion
```